onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorLayerNormalization.cpp

```diff
@@ -258,7 +260,8 @@ void CALLBACK QueryLayerNormalization(IMLOperatorSupportQueryContextPrivate* con
     *isSupported = context->GetOutputCount() == 1;
 }
 
-DML_OP_DEFINE_CREATION_FUNCTION(LayerNormalization, DmlOperatorLayerNormalization);
-DML_OP_DEFINE_CREATION_FUNCTION(LayerNormalization17, DmlOperatorLayerNormalization);
+DML_OP_DEFINE_CREATION_FUNCTION(LayerNormalization, DmlOperatorLayerNormalization<false>);
+DML_OP_DEFINE_CREATION_FUNCTION(LayerNormalization17, DmlOperatorLayerNormalization<false>);
+DML_OP_DEFINE_CREATION_FUNCTION(SimplifiedLayerNormalization, DmlOperatorLayerNormalization<true>);
```

Comment:
where is `SimplifiedLayerNormalization` from? I just see `SkipSimplifiedLayernormalization` here https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#commicrosoftskipsimplifiedlayernormalization, but no contrib op `SimplifiedLayerNormalization` nor here https://github.com/onnx/onnx/blob/main/docs/Operators.md#SimplifiedLayerNormalization.

Reply:
It took me a while to realize that they just don't have docs for it :/ I'm not sure why it doesn't get generated

https://github.com/microsoft/onnxruntime/blob/bb1972264b89261e98d438367eb54d97eea52c12/onnxruntime/core/graph/contrib_ops/contrib_defs.cc#L2966